Cartfile.resolved

```diff
@@ -1,5 +1,5 @@
-github "AliSoftware/OHHTTPStubs" "9.1.0"
-github "Quick/Nimble" "v12.0.0"
+github "Quick/Nimble" "v12.0.1"
 github "Quick/Quick" "v6.1.0"
+github "asana/OHHTTPStubs" "9.1.0-asana"
```

Comment:
[OHHTTPStubs](https://github.com/AliSoftware/OHHTTPStubs) no longer compiles with Xcode 14.3+. This library seems to no longer be maintained; it might be necessary to replace it altogether in the future.
The Asana fork includes the changes for Xcode 14.3+, which suffices for the time being.